user-interface/src/data-verification/ConsolidationOrderModal.test.tsx

```diff
@@ -140,7 +152,13 @@ describe('ConsolidationOrderModalComponent', () => {
     await waitFor(() => {
       expect(modal).toHaveClass('is-hidden');
     });
-    expect(onConfirmSpy).toHaveBeenCalled();
+
+    expect(onConfirmSpy).toHaveBeenCalledWith({
+      status: 'approved',
+      courtDivision: undefined,
+      leadCaseId: `${courts[0].courtDivision}-11-11111`,
```

Comment:
I think we should use some better constants in this test. We've defined `11-11111` in an array and then again in a const, but then use it again here literally.